swarm/api/http/test_server.go

```diff
@@ -33,44 +33,45 @@ type TestServer interface {
 }
 
 func NewTestSwarmServer(t *testing.T, serverFunc func(*api.API) TestServer, resolver api.Resolver) *TestSwarmServer {
-	dir, err := ioutil.TempDir("", "swarm-storage-test")
+	swarmDir, err := ioutil.TempDir("", "swarm-storage-test")
```

Comment:
I'm not sure if I see the reasoning behind most of the changes in this file. Except for a missing cleanup function most of it looks like just whitespace and variable name juggling. I usually try to avoid introducing these kind of deltas into PRs since its just more strain for reviewing and here the benefit of the change is not really obvious to me (it doesn't make the code more readable)

Reply:
The important part is adding `fileStore.Close()` to the cleanup closure. Now that is pointed out to me, I agree and see that the other changes hide the important change. That's bad.

However, with the added note that readability is subjective, I disagree that making these variable name changes did not carry value. 
- As I changed for example `rh` - I still cannot resolve that abbreviation -  to `feeds` what is easier to remember for me as it is pronounceable and also carries information.
- I also changed `storeparams` to `storeParams` as 'storage parameters' is two words and in Go is we use camel case naming. So the latter just a change to confirm with Go idioms.

Considering the above, next time I'll try to separate the refactoring into a separate commit. I hope that's sufficient to keep you happy. Otherwise, please, elaborate what problem you see with my new variable names and why they are not better than the previous ones.